src/languages/mathematica.js

```diff
@@ -6,61 +6,118 @@ Website: https://www.wolfram.com/mathematica/
+    begin: /\\\[/,
+    end: /[$a-zA-Z][$a-zA-Z0-9]+]/
```

Comment:
See note above regarding using just `begin` for simple matches.

Reply:
Again, it was the version that worked and I'm happy to get a helping hand if this can be simplified.

Reply:
It's possible I don't undersatnd what you're matching but if it's a SINGLE unit then:

```
    begin: /\\\[[$a-zA-Z][$a-zA-Z0-9]+]/,
```

Should work equally well.

Reply:
I think I know what you're trying to solve now, but just give me a code example and I'll take a closer look.